view/rooms/app.js

```diff
@@ -906,11 +906,11 @@ function closePagingModal() {
     'style',
     'display:none;'
   );
-  alert('Thanks, IMC team was paged and it’s working hard to fix the issue');
 }
 
 function pageImcTeam() {
   closePagingModal();
+  alert('Thanks, IMC team was paged and it’s working hard to fix the issue');
```

Comment:
Should the alert be after the post request? @diogo-pessoa

Reply:
yeah, I can move that.